blockchains/eth/eth_worker.js

```diff
@@ -142,6 +143,9 @@ class ETHWorker extends BaseWorker {
 
     for (const block of blocks) {
       const decoded_transactions = this.feesDecoder.getFeesFromTransactionsInBlock(block, receipts);
+      const blockNumber = block.transactions.length > 0 ? this.web3Wrapper.parseHexToNumber(block.transactions.blockNumber) : 0;
```

Comment:
is there a better way to get block number?

Reply:
Should be also available as `block.number`

Reply:
fixed